README.rst

```diff
@@ -1,38 +1,52 @@
+===================
 CDAP Ingest Project
 ===================
 
-Various ways to ingest data into CDAP. 
+Various ways to ingest data into the Cask Data Application Platform (CDAP).
+
 
 Build all packages
 ------------------
-mvn clean package 
 
-Build rpm for packages (File tailer and DropZone)
-----------------------
-mvn clean package -P rpm
+::
+
+  mvn clean package 
+
+
+Build rpm for packages (File Tailer and DropZone)
+-------------------------------------------------
+
+::
+
+  mvn clean package -P rpm
+
+
+Build debian for packages (File Tailer and DropZone)
+----------------------------------------------------
+
+::
+
+  mvn clean package -P deb
 
-Build debian for packages (File tailer and DropZone)
-----------------------
-mvn clean package -P deb
 
-For details, please see the README.md file in each submodule.
+For details, please see the README file in each submodule.
```

Comment:
If you are under windows and the file extensions are hidden, you won't see it...but...